includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -484,8 +499,13 @@ private function getSitenameData() {
 	}
 
 	private function getVerticalData() {
+		// fandom has no set vertical
+		if ( $this->product === self::PRODUCT_FANDOMS ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)